test/test_astartes.py

```diff
@@ -412,9 +412,18 @@ def test_train_test_split(self):
             )
 
     def test_return_indices(self):
-        """Test the ability to return the indices and not the values."""
+        """Test the ability to return the indices and the values."""
         with self.assertWarns(ImperfectSplittingWarning):
-            (indices_train, indices_test,) = train_test_split(
+            (
+                train_X,
+                test_X,
+                train_y,
+                test_y,
+                train_labels,
+                test_labels,
+                indices_train,
+                indices_test,
```

Comment:
For consistency, could we please update these variable names to `train_indices` and `test_indices` and `val_indices` below as well?